.eslintrc.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
             cases: {
               pascalCase: true
             },
-            ignore: [/.*\.stories.tsx$/, /.*\.test\.tsx$/]
+            ignore: [/.*\.stories.tsx$/, /.*\.test\.tsx$/, /.*use.*\.tsx$/]
```

Comment:
What's the reasoning for adding this? I'm not seeing any new files with this syntax 🤔

Reply:
`useDragAndDrop.tsx` there is some `jsx in that file